docs/development.md

```diff
@@ -182,20 +182,20 @@ Requirements:
  - A `minishift` or OpenShift environment with Service Catalog enabled
  - `odo` and `oc` binaries in $PATH
 
-To deploy an e2e test:
+To deploy an test availaible in test the suite, one can use a variable `ODO_TEST_SCENARIO`. No use of this variable execute the entire test suite.
 
 ```sh
 # The entire suite
 make test-e2e
 
 # Just the main tests
-make test-main-e2e
+make test-e2e ODO_TEST_SCENARIO="odoe2e"
 
 # Just component tests
-make test-cmp-e2e
+make test-e2e ODO_TEST_SCENARIO="test-cmp-e2e"
```

Comment:
Passed value doesn't match actual used value: it should be `odoCmpE2e`.

Reply:
It was completely out of my mind. Thanks for pointing it out

Reply:
Done